app/src/main/java/com/github/se/project/ui/profile/EditTutorProfile.kt

```diff
@@ -0,0 +1,219 @@
+package com.github.se.project.ui.profile
+
```

Comment:
Is this and EditTutorSchedule the same code as the one in the branch EditProfileTutor? We should maybe first merge my branch then this one.